### PR TITLE
config_tools: update gpu passthru doc

### PR DESCRIPTION
**Pull request by Kunhui-Li**
add "i915.modeset=0" in kernel cmd line parameter for
allowing users to access a Ubuntu desktop.

Signed-off-by: Kunhui-Li <kunhuix.li@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/tutorials/gpu-passthru.rst [hunk 1]
```diff
@@ -61,6 +61,15 @@ The example extends the information provided in the :ref:`gsg`.
 #. In the ACRN Configurator, create a shared scenario with a Service VM and one
    post-launched User VM.
 
+#. Select the Service VM and go to **Basic Parameters > Kernel
+   command-line parameters**.
+
+#. Append ``i915.modeset=0`` in **Kernel command-line parameters**.
+
+   .. image:: images/kernel_cmd_line.png
```
Comment (dbkinder) [inline doc/tutorials/gpu-passthru.rst:69]: There should not be red squiggly underlines in the picture.  Is this an actual screen shot from the configurator?

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/tutorials/gpu-passthru.rst [hunk 2]
```diff
@@ -61,6 +61,15 @@ The example extends the information provided in the :ref:`gsg`.
 #. In the ACRN Configurator, create a shared scenario with a Service VM and one
    post-launched User VM.
 
+#. Select the Service VM and go to **Basic Parameters > Kernel
+   command-line parameters**.
+
+#. Append ``i915.modeset=0`` in **Kernel command-line parameters**.
```
Comment (gvancuts) [inline doc/tutorials/gpu-passthru.rst:67]: Can we add a (short) explanation as to what that does and why we need it?

We could even add this one to this doc ;-) https://projectacrn.github.io/latest/user-guides/kernel-parameters.html

File: doc/tutorials/gpu-passthru.rst (hunk 1 above)
Comment (Kunhui-Li) [reply doc/tutorials/gpu-passthru.rst:69]: Yes, this is an actual screen shot from the configurator.

Comment (Kunhui-Li) [reply doc/tutorials/gpu-passthru.rst:69]: Will update the picture.

Comment (Kunhui-Li) [reply doc/tutorials/gpu-passthru.rst:70]: updated.

File: doc/tutorials/gpu-passthru.rst (hunk 2 above)
Comment (Kunhui-Li) [reply doc/tutorials/gpu-passthru.rst]: updated.

**Review by dbkinder — APPROVED** [review]
LGTM, thanks.

**Pull request merged**